msteams-platform/TOC.yml

```diff
@@ -386,6 +386,8 @@
       href: concepts/deploy-and-publish/appsource/post-publish/overview.md
     - name: Application Certification program
       href: concepts/deploy-and-publish/appsource/post-publish/application-certification.md
+  - name: Publish purchasable web app 
```

Comment:
Change to "Publish purchasable web apps" for consistency with the rest of the content.